Swift VectorBoolean/VectorBoolean/BezierExtensions/NSBezierPath_Boolean.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  NSBezierPath_Boolean.swift
+//  Swift VectorBoolean for macOS
+//
+//  Based on NSBezierPath+Boolean - Created by Andrew Finnell on 5/31/11.
+//  Copyright 2011 Fortunate Bear, LLC. All rights reserved.
+
+//  Based on UIBezierPath_Boolean - Created by Leslie Titze on 2015-05-19.
+//  Copyright (c) 2015 Leslie Titze. All rights reserved.
+
+//  Created by April Noren on 2017-08-28.
+//  Copyright (c) 2017 April Noren. All rights reserved.
+
+extension NSBezierPath {
+
+  func fb_union(_ path: NSBezierPath) -> NSBezierPath {
+    let thisGraph = FBBezierGraph(path: self)
+    let otherGraph = FBBezierGraph(path: path)
+    let resultGraph = thisGraph.unionWithBezierGraph(otherGraph)!
```

Comment:
force cast! nooo

we should either handle that and return an optional NSBezierPath instead of risking a crash.

Reply:
So turn that line into an `if let`?  I could also return the path appended if that fails, but I guess it would probably be best to just return an optional that way the client can check if it failed.

Reply:
yeah, should probably just return an optional